tools/ts/ast_util.ts

```diff
@@ -0,0 +1,237 @@
+import { relative } from "path";
+import { readFileSync } from "fs";
+import Project, {
+  ExportDeclaration,
```

Comment:
What does this syntax mean? Why is `Project` not inside the braces?

Reply:
FWIW, I've been thinking to move `Project` to a named export and not have a default export. It's the default export of the library right now.

Reply:
And it is supported ES6 import syntax.  It is default import and then named imports.  You can blame TC39 for default.  Something that sounded good on paper...

Reply:
`Project` is also a named export, I will convert to that.

Reply:
oops, it isn't... DOH!

Reply:
I'll do a minor revision that adds it as a named export tonight. I'm not a fan of default imports/exports either.